services/blockchain-indexer/shared/dataService/business/dynamicReward/defaultReward.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const { requestConnector } = require('../../../utils/request');
+
+const { getRewardTokenID } = require('./constants');
+
+const getDefaultRewardAtHeight = async (params) => {
+	const response = {
+		data: {},
+		meta: {},
+	};
+	const defaultReward = await requestConnector('getDefaultRewardAtHeight', { height: params.height });
+	const rewardTokenID = await getRewardTokenID();
+
+	response.data = {
+		defaultReward: defaultReward.error ? null : defaultReward,
```

Comment:
Was this not supposed to be like this?
```
		defaultReward: defaultReward.reward,
```

Reply:
Sorry, all changes were not brought to this branch. Hence the confusion, updated base branch